janusgraph-test/src/main/java/org/janusgraph/graphdb/JanusGraphPerformanceMemoryTest.java

```diff
@@ -45,6 +50,35 @@
     @Rule
     public TestRule benchmark = JUnitBenchmarkProvider.get();
 
+    @Test
+    void labelRetrivial() {
```

Comment:
What is this test testing? It has no asserts  and it probably also passes without your changes. Did you only use it during development?

Reply:
This test demonstrate performance improvement

How these type of test use assertions? One of the way it performance tests must executed in less time than referenced values for latest released version for same env

Reply:
@farodin91 mentioned some time ago that we could use a dedicated library for these kind of performance tests, but I think that we don't have anything like that right now.

I'm not sure what we want to do until then with tests like these, but since there are already similar tests in this class as you pointed out, we can also add this from my side. Then we can later hopefully easily port it to a framework where a certain performance threshold is enforced.